README.md

```diff
@@ -1,2 +1,60 @@
 # control_library
-control and dsp algorithms for owntech converters
+
+Control and dsp algorithms for owntech converters
+
+This library has been integrated and tested on [TWIST](https://github.com/owntech-foundation/TWIST) converters using [SPIN](https://github.com/owntech-foundation/SPIN) controller.
+
+The integration is based on the dedicated [OwnTech Power API](https://github.com/owntech-foundation/Core).
+
+__Remarks__: The library is currently not optimized to be (we hope) easier to read.
+
+## Features
+
+The library provides the following functionalities mainly through C++ objects :
+ * `Pid()`: standard form of the PID regulator.
+ * `Pr()` : Proportional Resonant regulator.
+ * `Rst()` : Discrete form of Polynomial regulator.
+ * `PllSinus()`: Software PLL (Phased Lock Loop)
+ * Digital filters: `LowPassFirstOrdreFilter()`, `NotchFilter()`
+
+`Pid()`, `Pr()` and `Rst()` inherit from the `Controller()` class which fix the same interface.
+
+
+## Installation
+
+We describe here the process to use it with the [Power API](https://github.com/owntech-foundation/Core) which has beed designed to use PlatformIO.
+
+The installation here recall the procedure of using a library in [PlatformIO](https://docs.platformio.org/en/latest/librarymanager/index.html).
+
+To use the library, you need to add the line above in the `platformio.ini` file.
+
+```ini
+lib_deps=
+    control_lib = https://github.com/owntech-foundation/control_library.git
+```
+
+You can find various examples in the [OwnTech examples library](https://github.com/owntech-foundation/examples) 
+
```

Comment:
Licence section missing

Reply:
There's a tab for that in Github, do I have to add it anyway  ? or put a link ?
![image](https://github.com/owntech-foundation/control_library/assets/160136699/34a45889-85ba-4f05-ae90-e4bdea951b7f)